Change mic-icon position

FridaLar opened the pull request:
The mic-icon is now located in the upper-left corner of the canvas. This should make it more obvious to the user that the mic is recording.

Issue Number: #338

FridaLar committed:
Change mic-icon position

The mic-icon is now located in the upper-left corner of the canvas.
This should make it more obvious to the user that the mic
is recording.

Issue Number: #338

runesee reviewed (changes requested):
Looks very good! Perhaps the microphone icon should start out full size instead, so that users immediately know it is depicting a mic.

FridaLar committed:
Change mic-icon animation

To better understand that the icon is a mic-symbol,
the animation starts in full size and shrinks, instead of
starting to grow from a small size where no one
sees that it is a mic

Issue Number: #338

runesee reviewed (approved):
Mic icon starts at full size now, good job
